iotilesensorgraph/iotile/sg/parser/parser_v1.py

```diff
@@ -112,6 +112,7 @@ def compile(self, model):
             statement.execute(self.sensor_graph, self._scope_stack)
 
         self.sensor_graph.initialize_remaining_constants()
+        self.sensor_graph.add_crc()
         self.sensor_graph.sort_nodes()
```

Comment:
I'm not sure, however we would likely want the configuration variable nodes to be added after all the nodes have been sorted. (Unsure exactly what this would do to the representation)

Reply:
Yeah it rearranges how the nodes are in its list. So that would affect it.

I will fix this to `add_crc` after sorting.